pyinterprod/interpro/contrib/gp.py

```diff
@@ -0,0 +1,40 @@
+import cx_Oracle
+
+
+def import_gp(cur: cx_Oracle.Cursor, file_path: str):
+    gp = []
+    with open(file_path, "rt") as fh:
+        for line in fh:
+            if line.startswith("AC"):
+                entry_ac = line.split()[1]
+            elif line.startswith("DE"):
+                description = line.split(" ", maxsplit=1)[1].strip()
```

Comment:
By default, `str.split()` splits on whitespaces, which is what we want here, so there is no need to pass a delimiter:

```suggestion
                description = line.split(maxsplit=1)[1]
```